app/assets/javascripts/dropdown.js

```diff
@@ -14,4 +14,10 @@ $(document).ready(function(){
       }
     }
   });
+
```

Comment:
I would create a separate dropdown.js for the filter dropdown. Like in language-picker.js